app/Nova/Resources/Wiki/Studio.php

```diff
@@ -129,6 +130,22 @@ public function fields(Request $request): array
                     ];
                 }),
 
+            BelongsToMany::make(__('nova.external_resources', 'Resources', ExternalResource::class))
```

Comment:
Erroneous closing parenthesis at the end of the line. There should be a closing parenthesis after 'nova.external_resources' as this is the translation key for the __() translation helper function.